### PR TITLE
431prep

### PR DESCRIPTION
**Pull request by trel**

### CODE REVIEW

File: docs/release_notes.md [hunk 1]
```diff
@@ -1,5 +1,147 @@
 #
 
+## 4.3.1
+
+Release Date: 2023-10-xx
```
Comment (trel) [inline docs/release_notes.md:5]: need to update

File: docs/plugins/pluggable_rule_engine.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ The iRODS Consortium currently supports the following rule engine plugins:
  - [C++ Logical Quotas Rule Engine Plugin](https://github.com/irods/irods_rule_engine_plugin_logical_quotas)
  - [C++ Metadata Guard Rule Engine Plugin](https://github.com/irods/irods_rule_engine_plugin_metadata_guard)
  - [C++ Storage Tiering Rule Engine Plugin](https://github.com/irods/irods_capability_storage_tiering)
-
+ - [C++ Indexing Rule Engine Plugin][https://github.com/irods/irods_capability_indexing]
```
Comment (korydraughn) [inline docs/plugins/pluggable_rule_engine.md:16]: Second pair of square brackets -> parens?

File: docs/release_notes.md [hunk 3]
```diff
@@ -1,5 +1,147 @@
...
+ - Add connection_pool features [#5060] [#5799] [#7130] [#7145] [#7287] [#7304] [#7314] [#7350]
+ - Add ticket administration library [#5716] [#7115] [#7179] [#7185] [#7211]
+ - Add server configuration reload on SIGHUP [#5816] [#5821] [#6569]
+ - Add client connection information to acPreConnect [#5985]
+ - Use boost::process to capture messages better [#6273]
+ - Expose CRON task intervals in advanced settings [#6342]
+ - Expose delay server and control plane timeout settings [#6370]
+ - Improved logging facilities [#6470] [#6493] [#6641] [#6642] [#6818] [#7048] [#7212]
+ - Add remote configuration reload to irods-grid [#6529]
+ - Add interactive_pam authentication plugin [#6576]
```
Comment (alanking) [inline docs/release_notes.md:34]: Not sure if it helps to clarify things or not, but we did not add a new plugin - just added support for it to `iinit`.

Comment (trel) [reply docs/release_notes.md:34]: ah - noted.  thanks.

**Comment by trel** [comment]
squashed, ready for eyeballs

**Review by alanking — APPROVED** [review]
Only thing I noticed is that everything is separated by 2 newlines in the 4.2.12 notes. Not sure if that's on purpose or whether it matters

**Comment by trel** [comment]
that matches the 4.2 way... tightened it a bit for 4.3.0... so kept them consistent.

thanks.

**Comment by trel** [comment]
adding #s and merging...

**Pull request merged**